js/scripts.js

```diff
@@ -1 +1,15 @@
- 
\ No newline at end of file
+var a = prompt('Wpisz wartość liczby a');
+var h = prompt('Wpisz wartość liczby h');
+
+function getTriangleArea(a, h) {
+	if ( (a <= 0) || (h <= 0) ) {
+		console.log("Nieprawidłowe dane");
+	} else ( (a > 0) && (h > 0) ) {
+		var result = a*h/2
+		return result
+		console.log( getTriangleArea(10,6) )
```

Comment:
po co ten log?

ta funkcja zwraca `result` czyli dany wynik i ten wynik w linijkach 13, 14 oraz 15 przypisujesz do zmiennej i to te zmienne powinieneś ewentualnie logować (w następnych linijkach)